logicrunner/goplugin/rpctypes/rpctypes.go

```diff
@@ -51,8 +51,8 @@ type DownCallConstructorReq struct {
 
 // DownCallConstructorResp is response from CallConstructor RPC in the runner
 type DownCallConstructorResp struct {
-	Ret              insolar.Arguments
-	ConstructorError string // gob can't serialize `error` thus we are using string here
+	Data []byte
+	Ret  insolar.Arguments
```

Comment:
Maybe rename this `Ret` to `Result`, for instance?
Sure it should have type `insolar.Arguments`?

Reply:
put on my cleanup list